src/main/java/appeng/integration/IntegrationType.java

```diff
@@ -39,24 +38,6 @@ public IIntegrationModule createInstance()
 
 	RC( IntegrationSide.BOTH, "Railcraft", "railcraft" ),
 
-	RF( IntegrationSide.BOTH, "RedstoneFlux Power - Tiles", "redstoneflux" )
-	{
-		@Override
-		public IIntegrationModule createInstance()
-		{
-			return new RFModule();
-		}
-	},
-
-	RFItem( IntegrationSide.BOTH, "RedstoneFlux Power - Items", "redstoneflux" )
-	{
-		@Override
-		public IIntegrationModule createInstance()
-		{
-			return new RFModule();
-		}
-	},
-
 	MFR( IntegrationSide.BOTH, "Mine Factory Reloaded", "minefactoryreloaded" ),
 
 	Waila( IntegrationSide.BOTH, "Waila", "waila" )
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS00115)